exampleCourse/questions/element/template/question.html

```diff
@@ -0,0 +1,38 @@
+<h5>This page shows has example uses of the <code>pl-template</code>
+  element, including uses of recursive templates.
+</h5>
+
+<hr>
+
+<pl-template file-name="outerTemplate.mustache" subdirectory="templates">
+  <pl-variable name="is-open">True</pl-variable>
+  <pl-variable name="problem-statement" parent-directory="question" subdirectory="serverFilesQuestion"
+    file-name="statement.html"></pl-variable>
+</pl-template>
+
+<hr>
+
+<pl-template file-name="cardTemplate.mustache" parent-directory="question" subdirectory="serverFilesQuestion">
+  <pl-variable name="header"><b>A bolded card header</b></pl-variable>
+</pl-template>
+
+<hr>
+
+<pl-template file-name="prompt_in_collapsed_section.mustache.html" parent-directory="server-files-course"
```

Comment:
Nit: while we can't safely run every single HTML file through Prettier yet (thanks mostly to my stupid `<markdown>` tag), it'd be nice if we could format new files with Prettier. https://prettier.io/playground is a lightweight way to do that.

Reply:
I've actually been using the VSCode plugin to do HTML formatting. I think that `pl-code` is another element that has issues with autoformatting.